### ISSUE TITLE
Can not close out of video preview in file manager NC 12

### ISSUE THREAD

**Issue by BrianEnders** [nextcloud/server#4931]
### Steps to reproduce
1. Upgrade to NC 12 Beta 4
2. Open File Manager to view a video file
3. Can not close video file. No way to exit it.

### Expected behaviour
An X should be present to close the video, or clicking outside the video player

### Actual behaviour
Expecting a way to close the video I am viewing. But clicking off the video does nothing.

### Server configuration

**Operating system**: Ubuntu 16.04 LTS

**Web server:** Apache

**Database:** MySQL

**PHP version:** 7

**Nextcloud version:** NC 12 Beta x - NC 12 GA

**Updated from an older Nextcloud/ownCloud or fresh install:** Updated

**Where did you install Nextcloud from:** NC 11.0.2

**Signing status:**
<details>
<summary>Signing status</summary>

```
Login as admin user into your Nextcloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.

No errors have been found.
```
</details>

**List of activated apps:**
<details>
<summary>App list</summary>
```
Activity 
2.5.2

Collaborative tags 
1.2.0

Comments 
1.2.0

Deleted files 
1.2.0

Federation 
1.2.0

File sharing 
1.4.0

First run wizard 
2.1

Gallery 
17.0.0

Log Reader 
2.0.0

Monitoring 
1.2.0

Nextcloud announcements 
1.1

Notifications 
2.0.0

Password policy 
1.2.2

PDF viewer 
1.1.1

Share by mail 
1.2.0

Text editor 
2.4.1

Theming 
1.3.0

Update notification 
1.2.0

Usage survey 
1.0.0

Video player 
1.1.0

Brute-force settings 
1.0.2

Calendar 
1.5.2

Contacts 
1.5.3

External sites 
2.0.2

Mail 
0.6.4

Passman 
2.1.2

Phone Sync 
1.12.0

Tasks 
0.9.5

Auditing / Logging 
1.2.0

Default encryption module 
1.6.0

External storage support 
1.3.0

External user support 
0.4

LDAP user and group backend 
1.2.1

Audio Player 
1.5.1

GitHub merge tracker 
0.0.17

Video calls 
2.0.0
```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder

or 

Insert your config.php content here. 
Make sure to remove all sensitive content such as passwords. (e.g. database password, passwordsalt, secret, smtp password, …)
```
</details>

**Are you using external storage, if yes which one:** local/smb/sftp/... (LOCAL)

**Are you using encryption:** yes/no (NO)

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/... (NO)

### Client configuration
**Browser:** Microsoft EDGE and Chrome

**Comment by ttbek** [comment]
Same behavior for me, not exactly the same config, but similar, released nextcloud 12 version.  My browser is Firefox instead.  OS is Ubuntu 16.04 (no Mate).  Client OS is Arch, Gnome desktop environment.  Backbutton also does not close the video, I got it to close only by refreshing or loading the page again.

**Comment by DDArn** [comment]
Same for me. I use debian stretch 64bit with nc12.0.0.
Browser is Firefox ESR 45.9.0 and other versions on different PCs. All the same issue.
There is no "X" to close the videoplayer. Only refreshing helps to close it.
I hope it is a small bug. NC11 does not have this.

**Comment by djermann** [comment]
Same here. I use debian 8 with Nextcloud 12.0.0. Closing the videoplayer is only possible through refreshing the page. Browser is Opera.

**Comment by jjcazau** [comment]
Debian 8 released v12 cant close out of video. Video also seems to be using default chrome html5 playback controls, i think i remember custom ones in v11. Only way is to refresh page, that takes you back to folder.

**Comment by JK74** [comment]
same issue from my side with U16.04 and NC12.0 ; I even have the issue that the nextcloud.mp4 does not play at all...

**Comment by kopasiak** [comment]
The same problem (Cannot close video preview) for me. U16.04 NC 12.0.0

**Comment by jasonbayton** [comment]
This definitely affects 12 stable too.

**Comment by jasonbayton** [comment]
https://github.com/nextcloud/files_videoplayer/issues/25#issuecomment-304231579

**Comment by knixx** [comment]
Same here. Unfortunately nobody seems to care.

**Comment by jasonbayton** [comment]
@knixx it's fixed in 12.0.1 if you read the linked comment....

**Comment by crickxson** [comment]
Hello, I'm using the 12.03, on Firefox 60.0 on Windowz and Ubuntu 16.04, and no close button. Is it hidden somewhere? Thanks.
